### PR TITLE
Issue 28 crawl3d

### PR DESCRIPTION
**Pull request by dmitryint**
again :)

### CODE REVIEW

File: rssconfig.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ class RssConfig(object):
             ("quality", """480p;720p;1080p""", "480p, 720p or 1080p", "720p"),
             ("pushbulletapi","str","Your Pushbullet-API key",""),
             ("hoster", """OBOOM;Uploaded;Share-Online;Zippyshare""", "Hoster to load from", "Uploaded"),
-            ("historical","bool","Use the search function in order to match older entries","False"),
+            ("historical","bool","Use the search function in order to match older entries","True"),
```
Comment (dmitryint) [inline rssconfig.py]: This variable is not processed correctly. I think it must be so.

**Comment by dmitryint** [comment]
Please check carefully:)

File: rssconfig.py (hunk 1 above)
Comment (rix1337) [inline rssconfig.py]: Wow, I would've not been able to spot this bug.

Was the True/False part not properly recognized as bool?

I would like to keep the historical setting off by default, but for checking purposes I will merge this PR as requested!

**Comment by rix1337** [comment]
Could we disable
Starting new HTTP connection (1): [...] from info log?

File: RSScrawler.py [hunk 2]
```diff
@@ -455,7 +454,7 @@ def send_package(self, title, link):
 
     pool = [
         MovieblogFeed(),
-        SJ(),
+        # SJ(),
```
Comment (dmitryint) [inline RSScrawler.py]: Sorry

**Pull request merged**